soe/src/sites/default/settings.php

```diff
@@ -9,7 +9,7 @@
   header('HTTP/1.0 301 Moved Permanently');
   $redirect_url = "https://" . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
   header("Location: " . check_plain($redirect_url));
-  drupal_exit();
+  exit();
```

Comment:
Hm. We have this code in a lot of places, methinks. Lemme look.

Reply:
Like [here](https://github.com/SU-SWS/acsf-cardinald7/blob/1.x/factory-hooks/pre-settings-php/force-https.php#L15)

Reply:
Yeah, we do. It shows up :alot: in the logs.

Reply:
Is that the "undefined function `cache_get()` message?

Reply:
No, that one has something to do with 404 pages. Acquia connector pings the site endpoint often causing it to happen, but also, normal traffic, seems to cause it.

Reply:
Acquia connector module is disabled AFAICT